rcore/power.c

```diff
@@ -57,7 +57,7 @@ void power_update_battery(void)
 {
     _bat_voltage = hw_power_get_bat_mv();
     _bat_pct = map_range(_bat_voltage, 2600, 3500, 0, 100);
-    SYS_LOG("PWR", APP_LOG_LEVEL_INFO, "VBAT %ldmV %d%%", _bat_voltage, _bat_pct);
```

Comment:
I guess this was annoying

Reply:
Yes :)
I also left my debug spews in for now